docs/api.md

```diff
@@ -589,8 +780,25 @@ level to access.
 One or more of the attribute names you have specified are not known.
 The `attributes` response field lists these.
 
+### Saved page not found
+
+A saved page cannot be found with the provided path in the user's
+store of saved pages.
+
+The path is either incorrectly formatted, misspelled or it is not
+stored in this user's account.
+
 #### Debugging steps
 
 - check that you don't have a typo in the attribute names
 - check that the attributes are defined in `config/user_attributes.yml`
 - check that you are running the latest version of account-api
+
+### Page cannot be saved
+
+An attempt was made to save the page with the provided page path, however it could not be persisted.
```

Comment:
"An attempt was made" is redundant, as all the endpoints attempt to do the thing they should.  Also this should mention the `errors` field.  How about

> The page you have specified could not be saved.  The `errors` response field lists the problems.